style.css

```diff
@@ -33,6 +33,27 @@ header {
   justify-content: space-around;
 }
 
+.div4 {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
```

Comment:
- [ ] I noticed that you used `display: flex` multiple times across your CSS stylesheet. It would be great if you find a way of reducing this code duplication to maintain the code as short as possible, making it easy to read. Please, consider fixing this! 🤓 🛠️ 

There are several ways of achieving this desired result, here's one of them as an example:

You can define a `flex` class with the `display: flex;` property on it.
```css
.flex {
  display: flex;
}
```

And then, use it whenever you need a flex container in your HTML elements:

```HTML
<div class="flex"> I'm a flex container </div>
```